plugins/modules/zos_data_set.py

```diff
@@ -109,7 +111,7 @@
     description:
       - The data set type to be used when creating a data set. (e.g C(pdse))
       - C(MEMBER) expects to be used with an existing partitioned data set.
-      - Choices are case-insensitive.
+      - Choices are case-sensitive.
```

Comment:
I think we should create a review item for playbook repository, just to correct any misleading examples.

Reply:
I see this is the case for many different options, so definitely needed.

Reply:
We got stuck, because we have to use choices, which are case-sensitive.  Let's mention it to Ketan during stand up, since he's going to check other things in the playbook repository.

Really, I'd like to 'invent' a way to make choices case-insensitive, but didn't see a safe way to do it.

Reply:
I've expanded the scope of this PR - https://github.com/IBM/z_ansible_collections_samples/pull/246